.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,62 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "bug: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        _The more information you share, the faster we can help you._
+        Prior to opening the issue, please make sure that you:
+        - Use English (EN/US) to communicate.
+        - Search the [open issues](https://github.com/hyperledger/firefly/issues) to avoid duplicating the issue.
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: What happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please disclose it privately via security@hyperledger.org
```

Comment:
This email might also be different - @tkuhrt @ryjones fyi

Reply:
security@lfdecentralizedtrust.org

Reply:
Thanks, updated